src/components/data-chart/index.tsx

```diff
@@ -0,0 +1,471 @@
+                            function (rect) {
+                              if (rect.getAttribute('fill') === '#007079') {
+                                rect.setAttribute(
+                                  'fill',
+                                  'url(#chart-gradient) #007079'
+                                );
+                              }
+                            }
+                          );
+                        });
+                        observer.observe(container, {
+                          childList: true,
+                          subtree: true,
+                        });
+                      },
+                    },
+                  ]}
+                  className="chart-area"
+                />
+                <svg
+                  style={{ width: 0, height: 0, position: 'absolute' }}
+                  aria-hidden="true"
+                  focusable="false"
+                >
+                  <linearGradient
+                    id="chart-gradient"
+                    x1="0"
+                    y1="0"
+                    x2="0"
+                    y2="1"
+                  >
+                    <stop offset="0%" stopColor="#007079" />
+                    <stop offset="87.5%" stopColor="#FFF" />
+                  </linearGradient>
+                </svg>
+                <Chart
+                  chartType="Timeline"
+                  rows={timelineDataPoints3}
+                  columns={[
+                    {
+                      type: 'string',
+                      id: 'Position',
+                    },
+                    {
+                      type: 'string',
+                      id: 'Name',
+                    },
+                    {
+                      type: 'string',
+                      //@ts-ignore
+                      role: 'tooltip',
+                      p: { html: true },
+                    },
+                    {
+                      type: 'date',
+                      id: 'Start',
+                    },
+                    {
+                      type: 'date',
+                      id: 'End',
+                    },
+                  ]}
+                  options={{
+                    timeline: {
+                      colorByRowLabel: false,
+                      showRowLabels: false,
+                      showBarLabels: false,
+                    },
+                    colors: timelineColors,
+                    hAxis: {
+                      format: 'LLLL dd, Y',
+                    },
+                    tooltip: {
+                      isHtml: true,
+                      trigger: 'focus',
+                    },
+                  }}
+                  className="chart-timeline"
+                />
+              </div>
+            </Dialog>
+          </Scrim>
+        )}
+      </>
+    );
+  } else {
+    return (
+      <>
+        <span>
+          <CircularProgress size={16} /> Loading
+        </span>
+      </>
```

Comment:
remove unnecessary Fragments from HTML
```suggestion
      <span>
        <CircularProgress size={16} /> Loading
      </span>
```